diff --git a/MANIFEST.in b/MANIFEST.in
index b667159..ba76378 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1 +1 @@
-graft src/calibreweb
+graft .
diff --git a/setup.cfg b/setup.cfg
index b10c282..9ce9d1e 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -34,8 +34,8 @@ keywords =
 python_requires = >=2.6
 
 [options.entry_points]
-console_scripts =
-  cps = calibreweb:main
+scripts =
+  cps = cps.py
 [options]
 include_package_data = True
 dependency_links = comicapi @ git+https://github.com/OzzieIsaacs/comicapi.git@5346716578b2843f54d522f44d01bc8d25001d24#egg=comicapi
diff --git a/setup.py b/setup.py
index 6bde221..d1bdfd4 100644
--- a/setup.py
+++ b/setup.py
@@ -40,7 +40,6 @@ def find_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 setup(
-    packages=find_packages("src"),
-    package_dir = {'': 'src'},
-    version=find_version("src", "calibreweb", "cps", "constants.py")
+    packages=['cps'],
+    version=find_version("cps", "constants.py")
 )
